src/utils.js

```diff
@@ -0,0 +1,72 @@
+import moment from 'moment';
+
+const MILLISECONDS = 1000;
+
+
+export const groupByDate = (list) => list.reduce((groups, item) => {
+  const date = moment(item.dt * MILLISECONDS).format('YYYY-MM-DD');
+  if (groups[date]) {
+    groups[date].push(item);
+  } else {
+    groups[date] = [item];
+  }
+  return groups;
+}, {});
+
+
+export const sortByDate = (hash) => {
+  const dates = Object.keys(hash).sort((current, next) => moment(current) - moment(next));
+  return dates.map(date => hash[date]);
```

Comment:
Codacy has a fix for the issue: [Expected parentheses around arrow function argument.](https://app.codacy.com/gh/megwali/punny-weather/pullRequest?prid=7693782)
```suggestion
  return dates.map((date) => hash[date]);
```